### ISSUE TITLE
Rightward overflow condition perceived by user as corrupt output.

### ISSUE THREAD

**Issue by dmullis** [grid-js/gridjs#295]
An example, showing milder symptoms, can be seen by resizing this example to be narrow enough that the right half of the right-hand column's contents are clipped: https://gridjs.io/docs/examples/row-buttons

An especially problematic case occurs when:
* pagination is disabled
* number of rows exceeds limits of window height, so table is clipped at bottom
* the right-hand column happens to contain reflowing text

A human reader attempts to scan text in the rightmost column from one line to next, and some sentences appear to be gibberish.

One workaround is to disable settings of two 'div' elements' "overflow" attributes, by forcefully modifying these CSS classes:
```
/* Make rightward overflow obvious.
   Testing shows both classes must be modified */
.gridjs-container {
    overflow: unset !important;
}
.gridjs-wrapper {
    overflow: unset !important;
}
```
With this, the user gets two strong visual cues that output has been clipped, independent of cell contents:

1. Square corners at right side of table frame.
2. Appearance of horizontal scroll bar.

The user then promptly understands how to expose the desired text: Either a) window resize or b) rightward scroll will suffice.

Note that the above workaround allows the valuable 'autoWidth' config option to remain enabled.

**Comment by afshinm** [comment]
Gotcha @dmullis. Thanks a lot for clearly explaining the issue.

I see your point about setting the overflow to `unset` but I'm not sure if that's the best solution. I think a better solution is to somehow enable the horizontal scrollbar for the inner container (or something similar to show that the table viewport is clipped)

**Comment by dmullis** [comment]
Yes, seems the outer window-scope scrollbar would remain problematic if, as one should assume, the table is embedded on a page with significant other content.  In my current case, there's nothing else, so it's tolerable.  

However, I belatedly note another weakness of my workaround: On initial page load, or forced refresh, the scrollbar appears, and table is wider than window by about 5%.  A manual resize by just that ~5% wider corrects the appearance to unclipped, no scroll bar.

**Comment by afshinm** [comment]
I tried a couple of things, this solution in particular: https://lea.verou.me/2012/04/background-attachment-local/ but not sure why it didn't work with `gridjs-wrapper` layer -- I think this issue only exist on mobile devices and not desktop? 

PR welcome.
